### ISSUE TITLE
Facing issue in accessing the Chime Connect Demo-Agent Portal

### ISSUE THREAD

**Issue by aartikal** [aws-samples/amazon-chime-sdk-amazon-connect-integration-demo#11]
Hi, 
As instructed I have built the agent portal on port 8080 and customer portal on port 3000 as port 9000 was unavailable.
However, I am facing some certificate error with 'Not secure' tag along with 'application refused to connect'

Attached the error screenshot.
Kindly advise.

![image](https://user-images.githubusercontent.com/110903158/183703570-4eea556b-9c9c-4b80-8bc9-ce3caf768fd8.png)

**Comment by jobijaco** [comment]
Hi team,
The certificate is not valid is appearing for me as well.

Attaching the screenshot of the Certificate for more info.
![Screenshot 2022-12-20 at 12 59 14 PM](https://user-images.githubusercontent.com/97499424/208608576-db1761e8-1bdd-4e1b-b3c6-67cfa83f6be7.png)

**Comment by devalevenkatesh** [comment]
Sorry for the delay in getting to this issue. This certificate issue happens when the application running at localhost:8080 is not able to connect to Amazon Connect agent CCP website. In this case, `SAMEORIGIN` `X-Frame-Options` error may also be seen in the browser console where the agent application keeps on trying to connect but Amazon Connect does not respond due to certificate issue.

To resolve this, open `https://<YOUR AMAZON CONNECT INSTANCE ALIAS>.awsapps.com/connect/ccp-v2/chat` in a separate tab in same window, login using the administrator you created during Amazon connect instance creation and again refresh the https://localhost:8080 tab.

Following this solved my issue. On a side note, I have updated the demo and have done minor enhancements from Amazon Chime SDK for JavaScript side. All changes can be seen in the PR description: https://github.com/aws-samples/amazon-chime-sdk-amazon-connect-integration-demo/pull/12

I tested the demo with those changes and everything works as expected now.
